README.md

```diff
@@ -20,12 +20,16 @@ The easiest and quickest way to get started with Analytics 2.0 is to [use it thr
 
 ## 💡 Using with Segment
 
+**There is a [React example repo](https://github.com/segmentio/react-example/tree/main/src/examples/analytics-quick-start) which outlines using the Segment snippet.**
+
 1. Create a javascript source at [Segment](https://app.segment.com) - new sources will automatically be using Analytics 2.0! Segment will automatically generate a snippet that you can add to your website. For more information visit our [documentation](https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/)).
 
 2. Start tracking!
 
 ## 💻 Using as an NPM package
 
+**There is a [React example repo](https://github.com/segmentio/react-example/tree/main/src/examples/analytics-package) which outlines using the Segment npm package.**
```

Comment:
I wonder if it makes more sense to create a `using React` subsection like we have for Vue and place this comment there.

I think we should probably still have a snippet that shows calling `AnalyticsBrowser.load()` under item 2 to get an instance of analytics since that's the bare minimum you'd need today to get started.